.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: 'User Reported Bug: {Insert reported bug here}'
+labels: bug
```

Comment:
user-visible? bug is not an helpful label

Reply:
Proposed `user-reported-bug`.

Other options could be:
user-visible-bug
user-report